app/plugin_api.go

```diff
@@ -650,6 +650,10 @@ func (api *PluginAPI) GetPluginStatus(id string) (*model.PluginStatus, *model.Ap
 	return api.app.GetPluginStatus(id)
 }
 
+func (api *PluginAPI) IsPluginCompatible(id string) (bool, *model.AppError) {
```

Comment:
I don't think we want to expose a new API method (across RPC) for this functionality, given it can be composed from existing calls. See proposal above for a different direction to consider.